File: python_examples/task6/problem_6.py
"""
    Find the difference between the sum of the squares of the first one hundred natural
    numbers and the square of the sum.
"""


def get_result(n: int) -> int:
    sum_of_square_natural_numbers = sum([v * v for v in range(1, n+1)])
    square_of_sum_natural_numbers = pow(sum([v for v in range(1, n+1)]), 2)
    return square_of_sum_natural_numbers - sum_of_square_natural_numbers


print(f'Sum of n natural numbers and square of the sum is: {get_result(100)}')

File: java-examples/src/com/szymczak/task6/Problem6.java
package com.szymczak.task6;

import java.util.stream.LongStream;

/*
    Find the difference between the sum of the squares of the first one hundred natural
    numbers and the square of the sum.
*/
public class Problem6 {
    public static void main(String[] args) {
        System.out.printf("Sum of n natural numbers and square of the sum is %d", getResult(100));
    }

    private static long getResult(int n) {
        long sum = LongStream.rangeClosed(1, n)
                .map(value -> value * value)
                .sum();
        long sumOfNNaturalNumbersWithSquare = (long) Math.pow(LongStream.rangeClosed(1, n)
                .sum(), 2);
        return sumOfNNaturalNumbersWithSquare - sum;
    }
}
